sdk/core/Azure.Core/src/Pipeline/LoggingPolicy.cs

```diff
@@ -14,19 +14,23 @@ namespace Azure.Core.Pipeline
 {
     internal class LoggingPolicy : HttpPipelinePolicy
     {
+
+        public LoggingPolicy(bool logContent)
+        {
+            _logContent = logContent;
+        }
+
         private const long DelayWarningThreshold = 3000; // 3000ms
         private static readonly long s_frequency = Stopwatch.Frequency;
         private static readonly HttpPipelineEventSource s_eventSource = HttpPipelineEventSource.Singleton;
+        private readonly bool _logContent;
 
-        public static readonly LoggingPolicy Shared = new LoggingPolicy();
```

Comment:
Is it worth having `SharedNoContent` and `SharedWithContent`?

Reply:
I suspect it would have more parameters as we go.

Reply:
I suspect it would have more parameters pretty soon